src/lib.rs

```diff
@@ -907,6 +907,35 @@ impl Log for Logger {
 
     fn log(&self, record: &Record) {
         if self.matches(record) {
+            match record.level() {
```

Comment:
just noting this is the addition we're making

Reply:
yes